.github/workflows/go-validate.yml

```diff
@@ -1,69 +1,80 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: MPL-2.0
 
-#
```

Comment:
This file is actually managed by Terraform, so refactoring it means we can no longer keep it updated via our Terraform example in an internal repo, while this is a bit ungraceful do you mind leaving this unchanged?

Reply:
Just this portion or the entire config? 

Can you replicate it to the internal repo for Terraform to pick it up - notice the actions seem lag a bit too.

Reply:
The whole workflow as I understand it, currently we're blocked on updating it but there is work planned to fix this.  FYI @mogrogan.

Is there a specific thing you needed to update? Like a specific dependency bump? It would make the Terraform config less prone to fail on apply if you just updated the specific dependencies if that is your concern.

Reply:
Thinking this a bit differently, this could be considered / addressed with reusable workflows from a shared repo for plugin related actions that are the same across projects.

```yaml
steps:
      - uses: hashicorp/packer-plugin-actions/.github/workflows/ reusable-action.yml@<releaseCommitHash>
```

Reply:
Absolutely, this would be a better way to handle this I agree,  it would be better if this logic wasn't hidden away with Terraform, speaking from my own experience on the Packer Azure plugin I've had similar issues wanting to update workflows.

I don't know when we would be able to prioritize the work on something like this, but I will raise this suggestion with the team though.